packages/shell-api/src/helpers.ts

```diff
@@ -605,7 +605,7 @@ export function processFindAndModifyOptions<BaseOptions extends FindOneAndReplac
   if ('new' in options) {
     options.returnDocument = options.new ? 'after' : 'before';
     delete options.returnOriginal;
-    delete options.returnNewDocument;
+    delete options.new;
```

Comment:
I think we’ll want to remove all three of `returnOriginal, returnNewDocument, new` in all of the conditionals here, right?

Reply:
@addaleax, at that point there cannot be `returnNewDocument` in the options. But I can add it for clarity and readability. See my new commit

Reply:
Ah yeah, you’re right. Thank you either way! :slightly_smiling_face: